src/main/resources/application.yml.example

```diff
@@ -3,13 +3,25 @@ spring:
     web-environment: false
 
 rebazer:
-  team: your_company
-  user: service_user
-  pass: dont_use_this_pass_at_home
-  workspace: ./rebazer-workspace
-  garbageCollectionCountdown: 20
-  repos:
-    - name: the_repo_name
-      branch: master
-    - name: the_other_repo_name
-      branch: master
+  workspace: ./rebazer-workspace # optional
+  garbageCollectionCountdown: 20 # optional
+  pollInterval: 60 # optional
+  hosts:
+    - type: BITBUCKET # possible values GITHUB or BITBUCKET
+      url: https://bitbucket.org/ # optional, default is type dependent (github.com, bitbucket.org)
+      teams:
+      - name: your_company
+        user: service_user # optional value, default is $name
+        pass: dont_use_this_pass_at_home
+        repos:
+        - name: the_repo_name
+          masterBranch: develop # optional value, default is "master"
+        - name: otherRepo
```

Comment:
Mix of underscore_style and camelCaseStyle

Reply:
done